tss-esapi/src/context/tpm_commands/enhanced_authorization_ea_commands.rs

```diff
@@ -141,7 +141,13 @@ impl Context {
         policy_session: PolicySession,
         digest_list: DigestList,
     ) -> Result<()> {
-        let digest_list = TPML_DIGEST::try_from(digest_list)?;
+        if 2 > digest_list.len() {
```

Comment:
Maybe that just me but this is hard to read. Why not just `...len <= 2`?

Reply:
```if digest_list.len() <= 2``` would not be correct. 2 is a valid number. And I am not so sure that ```if digest_list.len() <= 1``` makes it more readable.

Reply:
Sorry, my mistake I was thinking about `if digest_list.len() < 2`. With the 2 in front it just looks like [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions) :)

Reply:
fixed